cdc_test.py

```diff
@@ -547,7 +547,7 @@ def _init_new_loading_node(self, ks_name, create_stmt, use_thrift=False):
         logger.debug('adding node')
         self.cluster.add(loading_node, is_seed=True)
         logger.debug('starting new node')
-        loading_node.start(wait_for_binary_proto=True)
+        loading_node.start(wait_for_binary_proto=120)
```

Comment:
instead of increasing timeout we could use some magical flag 
to skip seed node2 seed connection checking (as node1 is down and node2 is also a seed).
this is new "safety guard" in C* 4.0, but I have not found any switch to turn it off...

I was considering to make timeout conditional for 4.0 but this would imo bring a little value with added code complexity